doc/faq.rst

```diff
@@ -90,6 +91,14 @@ How do I make a scatterplot of columns from two different catalogs?
 -------------------------------------------------------------------
 :ref:`Merge <merging>` the two datasets into a single object.
 
+How do I make a scatterplot between two images?
+-------------------------------------------------------------------
+:ref:`Merge <merging>` the two datasets into a single object, like you do for catalogs. This will only work if the two images have exactly the same shape (although it does not check if the WCS is the same).  You can then drop the merged object with the **Image Viewer**, and do this a second time for which you will select the other *Attribute* (on lower left) to be visible. For a third one you select the **Scatter Plot**. You can then select regions in any of the three plots, and the corresponding selections in the other plots will be highlighted. Another interesting way to visualize your two related images is drop a 4th object using **Image Viewer**, but this time select the RGB instead of Monochrome, and put each image into one of the colors. Right mouse down will change the *Contrast* selected color after you have selected the *bias/contrast button* on the top row. Obviously with three images  this is a great way to create a colorful RGB image.
```

Comment:
Including a screenshot of your two image + scatterplot setup would be helpful here. Also I'd prefer to standardize some of the wording:

"Drop the merged object with the Image Viewer..." -> "Drag the merged dataset into the visualization area to view an image of the first attribute. Repeat a second time to view the second image, and finally create a scatterplot comparing the image intensities at each pixel"